### PR TITLE
[new] change to deploy into mainnet

### PR DESCRIPTION
**Pull request by GustavoRSSilva**

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -23,7 +23,7 @@ APPCOINSIAB_CONTRACT_NAME=appcoinsiab
 IAB_DEVELOPMENT_ADDRESS=address_appcoinsiab_development
```
Comment (DioPires) [inline .env.example:23]: #### Tests:
```diff 
- 1 failing
```
```diff 
+ 48 passing
```
#### Test coverage: 
78.47 %

Comment (DioPires) [inline .env.example:23]: #### Tests:
```diff 
- 1 failing
```
```diff 
+ 48 passing
```
#### Test coverage: 
78.47 %

Comment (DioPires) [inline .env.example:23]: #### Tests:
```diff 
- 1 failing
```
```diff 
+ 48 passing
```
#### Test coverage: 
78.47 %

**Review by tcrosado — APPROVED** [review]

File: .env.example (hunk 1 above)
Comment (DioPires) [inline .env.example:23]: #### Tests:
```diff 
- 1 failing
```
```diff 
+ 48 passing
```
#### Test coverage: 
78.47 %

**Pull request merged**